### DISCUSSION TITLE
Remote (Innr RC 110) controlling all lights

### DISCUSSION THREAD

**Discussion by LouisMT** [Koenkk/zigbee2mqtt#8280] [Devices]
I just got a new Zigbee remote: an Innr RC 110. After pairing it with zigbee2mqtt, I can see the button presses in the log. However, the remote is also controlling all Zigbee lights. For example, when I press the scene 1 button, I can see this event in zigbee2mqtt, but also all Zigbee lights in my network dim to minimum brightness.

I don't want the remote to control my lights directly. I just want to receive the button presses in Home Assistant. In the docs, I've read about Zigbee binding. But it's not clear for me if I can turn this off, and how.

The binding page looks like this in the web UI:
![Screenshot from 2021-08-06 21-09-13](https://user-images.githubusercontent.com/1782761/128560136-88d0fba0-1d7e-42f0-8a05-c2b39c4464dc.png)

Anyone knows what's going on?

**Comment by Larsen-Locke** [comment]
I can confirm that this remote is just not working correctly in scenes mode. Although completely reseted and paired for it directly switches on my innr plugs when pressing the power button (zigbee-stick pulled out). This cannot be intended behavior so it must be faulty firmware.
In lights-mode everything works correct now (but reseted and repaired several times)
